### DISCUSSION TITLE
Closing discussions

### DISCUSSION THREAD

**Discussion by TonyGravagno** [RealRaven2000/FiltaQuilla#292] [General]
Perhaps discussions might remain open until mutually finalized? 🙏

**Comment by TonyGravagno** [comment]
After posting this discussion I read the following comment, which is **exactly** the reason why I posted this discussion. 😁 Thanks.

_Originally posted by @RealRaven2000 in https://github.com/RealRaven2000/FiltaQuilla/discussions/280#discussioncomment-10166545_

> Just a minor note on closing tickets - it is alright with this one as it is so minor and doesn‘t really affect many users, but with other things like obvious bugs and new features i like to keep my issues open until i release an official version on ATN. Partly because it is part of my workflow but mainly so that users who have the same issue see that it‘s actively worked on and so duplicates are also avoided. Closed issues just vanish from the main issues page because it‘s by default filtered to only show open ones.

**Comment by RealRaven2000** [comment]
I think discussion may remain open indefinitely as long as they are deemed important - the comment applied to **_issues_**, which usually affect code changes (bug fixes, function implementation) - and I like to keep these in sync with the latest releases.

**Comment by RealRaven2000** [comment]
changed emphasis on previous comment to read **_issues,_** which makes a big difference in meaning. Issues are essentially tracking tickets that can be linked to a release and code commits.
